File: python_agentsEnd/config/cfg_agent.py
#######################################################################
# PACKAGE IMPORTS
#######################################################################
INTERFACES = {
    'IPLUGD': 0,
    'OPAL_RT': 1
}
CONFIG = {
    'AGENT_NAMESPACE': 'agent_namespace',
    'IS_EXPERIMENTAL': False,
    'INTERFACE': INTERFACES['OPAL_RT'],
    'EXP_STREAM_FREQ': 1000, # in ms
    'IS_LOCALHOST': True, # make this false to open server on local ip instead of localhost
    'SENSOR_STREAM_FREQ': 1000 # in ms. Does not take effect if IS_EXPERIMENTAL is True
}

File: agentsEnd/config/cfg_agent.js
/******************************************************
 * Contains backend configuration
 *****************************************************/
export default {
    AGENT_NAMESPACE: "agent_namespace",
    IS_EXPERIMENTAL: true,
    IS_EXP_SYSTEM_PRESENT: false,//false
    EXP_SYSTEM_PROTOCOL: "UDP",
    SYSTEM_MULTICAST_IP: "224.0.0.114",
    SYSTEM_SOCKET_PORT: 5556,
    SYSTEM_LOCAL_PORT: 5556,
    EXP_STREAM_FREQ: 100, // in ms
    IS_LOCALHOST: true // make this false to open server on local ip instead of localhost
} 